File: Postfix/Postfix.py
exp = input('Enter a postfix expression: ')
elems = exp.split()

stack = []
for elem in elems:
    try:
        num = int(elem)
        stack.append(num)
    except:
        b = stack.pop()
        a = stack.pop()
        if elem == '+': stack.append(a + b)
        elif elem == '-': stack.append(a - b)
        elif elem == '*': stack.append(a * b)
        elif elem == '/': stack.append(a / b)

print(stack[0])

File: Postfix/Postfix.cs
using System;
using System.Collections.Generic;

public class Program
{
    public static void Main()
    {
        Console.Write("Enter a postfix expression: ");
        string exp = Console.ReadLine();
        string[] elems = exp.Split();

        Stack<string> stack = new Stack<string>();
        foreach (String elem in elems)
        {
            try
            {
                int num = int.Parse(elem);
                stack.Push(num.ToString());
            }
            catch
            {
                int b = int.Parse(stack.Pop());
                int a = int.Parse(stack.Pop());
                if (elem == "+") stack.Push((a + b).ToString());
                else if (elem == "-") stack.Push((a - b).ToString());
                else if (elem == "*") stack.Push((a * b).ToString());
                else if (elem == "/") stack.Push((a / b).ToString());
            }
        }
        Console.Write(stack.Peek());
    }
}
